sky/provision/fluidstack/instance.py

```diff
@@ -298,7 +298,7 @@ def query_instances(
         'pending': status_lib.ClusterStatus.INIT,
         'stopped': status_lib.ClusterStatus.STOPPED,
         'running': status_lib.ClusterStatus.UP,
-        'unhealthy': status_lib.ClusterStatus.INIT,
+        'failed': status_lib.ClusterStatus.INIT,
```

Comment:
Why changing this? Should we add a new entry instead?

Reply:
"unhealthy" not used anymore, refactored to "failed".

Reply:
For `H100-NVLink` not showing in the list of GPUs, recall that the catalog is fetched from the Skypilot catalog repository, which itself is generated from code currently in the main branch of Skypilot. The code in the main branch does not contain the new mapping, as such the GPU will not show. 

To view this new GPU locally, we need to fetch the catalog from FluidStack using the code from the [ forked repo](https://github.com/fluidstackio/skypilot/tree/fluidstack-nvlink-gpus).

```
python3 sky/clouds/service_catalog/data_fetchers/fetch_fluidstack.py
cp fluidstack/vms.csv ~/.sky/catalogs/v5/fluidstack/vms.csv 
sky show-gpus --cloud fluidstack -a
```
 
 We also need to add the FluidStack API key `~/.fluidstack/api_key` obtainable from the [dashboard](https://dashboard.fluidstack.io) prior to fetching the catalog from the FluidStack API.